server/server.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const path = require('path');
+const db = require('./db/config.js');
+// const env = require('dotenv').config(); <<<< NEED TO IMPLEMENT THIS
+
+const port = process.env.PORT || 3003;
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '../dist')));
+
+app.get('/product_info', (req, res) => {
+  db.query(`SELECT * FROM products WHERE sku = ${Math.floor(Math.random() * 50)}`, (err, response) => {
```

Comment:
You can modularize this a bit by wrapping the query in a helper function and invoking it here.